### DISCUSSION TITLE
Open links in background

### DISCUSSION THREAD

**Discussion by AbeEstrada** [wezterm/wezterm#3504] [Q&A] [answered]
Is there a way to make the links or other commands to open in background?

I use `newsboat` and `tut`, sometimes I need to open Safari or QuickTime from those TUI apps and would be nice to open those apps in background to keep using the WezTerm and when I have time to read the links go back to the browser.

I couldn't find any reference or setting.

Thanks.

**Comment by nekowinston** [comment, marked as answer]
There's the event [`open-uri`](https://wezfurlong.org/wezterm/config/lua/window-events/open-uri.html), which interjects how this is handled.

Looking at `man open` on Darwin, the manual page says:
```
     -g  Do not bring the application to the foreground.

     -j  Launches the app hidden.
```
which means, this *should* work: 
```lua
wezterm.on("open-uri", function(window, pane, uri)
  wezterm.run_child_process({ "open", "-gj", uri })
end)
```

But this actually still focuses the browser for me. I've searched for why this happens [[1]](https://apple.stackexchange.com/questions/445895) [[2]](https://apple.stackexchange.com/questions/260254), it seems this *used* to work, but doesn't anymore?

I hope these are at least useful pointers, if you end up figuring it out.

**Reply by AbeEstrada** [reply]
Thank you very much, it works for all my other apps but WezTerm but gives me an idea what to search.
